lib/os_allocation.c

```diff
@@ -1490,7 +1490,7 @@ os_l2ca_get_min_cbm_bits(unsigned *min_cbm_bits)
 }
 
 int
-os_mba_set(const unsigned socket,
+os_mba_set(const unsigned mba_id,
```

Comment:
Please update header file

Reply:
yes. again.. These are taken care in further patches.

Reply:
pulled couple more patches to address it. thanks.